test/unit/math/prim/scal/prob/bernoulli_test.cpp

```diff
@@ -40,3 +42,33 @@ TEST(ProbDistributionsBernoulli, chiSquareGoodnessFitTest) {
 
   EXPECT_TRUE(chi < quantile(complement(mydist, 1e-6)));
 }
+
+
+TEST(ProbDistributionsBernoulli, logitChiSquareGoodnessFitTest) {
+  boost::random::mt19937 rng;
+  int N = 10000;
+  boost::math::chi_squared mydist(1);
+ 
+  double parameter = -0.5;
+  double prob = stan::math::inv_logit(-0.5);
+  int bin[2] = {0, 0};
+  double expect [2] = {N * (1 - prob), N * prob};
+
+  int count = 0;
+
+  while (count < N) {
+    int a = stan::math::bernoulli_logit_rng(parameter,rng);
+    if(a == 1)
+      ++bin[1];
+    else
+      ++bin[0];
+    count++;
+   }
+
+  double chi = 0;
```

Comment:
Not necessary, but it'd be great if you could convert to `std::vector` and pull this out into a utility test function

```
void chi_square_test(const std::vector<int>& expect, const std::vector<int>& bin);
```
that would declare the `mydist` variable internally and run the `EXPECT_TRUE`.  Then we could have another issue to clean up all the other rngs which probably do it this way.  We've created almost all of our functions that operate on sequences using `std::vector` rather than using arrays or pointers mainly for memory encapsulation.  It makes the second init harder and the first one easier.